pkg/config/dynamic/tcp_config.go

```diff
@@ -72,8 +73,9 @@ type TCPServersLoadBalancer struct {
 	// connection, to close the reading capability as well, hence fully terminating the
 	// connection. It is a duration in milliseconds, defaulting to 100. A negative value
 	// means an infinite deadline (i.e. the reading capability is never closed).
-	TerminationDelay *int        `json:"terminationDelay,omitempty" toml:"terminationDelay,omitempty" yaml:"terminationDelay,omitempty"`
-	Servers          []TCPServer `json:"servers,omitempty" toml:"servers,omitempty" yaml:"servers,omitempty" label-slice-as-struct:"server"`
+	TerminationDelay     *int        `json:"terminationDelay,omitempty" toml:"terminationDelay,omitempty" yaml:"terminationDelay,omitempty"`
+	ProxyProtocolVersion *int        `json:"proxyProtocolVersion,omitempty" toml:"proxyProtocolVersion,omitempty" yaml:"proxyProtocolVersion,omitempty"`
```

Comment:
Could you replace the `ProxyProtocolVersion` type by a string? Having an integers as a version can be harmful if future versions doesn't follow this pattern. There'll be no need for a pointer here, its zero value would simply mean to not enable proxy protocol. And for the versions, `"1"` and `"2"` will be fine.

Reply:
my first solution had no pointer and default value 0, this resulted in changing so many test and integration files that i dropped this solution after changing so many lines of testcode. Followed your guide that the change should be reviewable.

Are you sure about string? There are two other *int values in same struct. 
In v2 spec the version size (the 13th byte) is only one byte

Reply:
please advice if it should be a string or int, i really recommend to keep the pointer.

Reply:
I tried to replace the `*string` by a `string` and I didn't found any problems with the tests. We would definitely prefer a `string` as a pointer is not useful in this case.

Reply:
Ok I will try tomorrow. Maybe the int was the problem.